templates/ttvars.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+## TT_PROJECTS - This is the root folder of your git-backed projects
+TT_PROJECTS=~/tmp
```

Comment:
This is really cool. I suppose this would give you the option to both:

  1. create two workspaces that share the same source code directories for equally named services
  2. create two workspaces that do not share the same source code directories for equally named services